Azure/sonic-utilities
# [CLI]Let speed 100M/2.5G can be correct displayed

#### What I did
Let speed 100M/2.5G can be correct displayed
#### How I did it
Parse the speed string:
The speed string length < len('1000'), display speed by unit 'M'
else display speed by 'G'

#### How to verify it
Real device verification.
Verify by the CLI cmd 'show interfaces status' output on the port which is configure with speed 100M
100M shall be shown in the Speed field

Verify by the CLI cmd 'show interfaces status' output on the port which is configured with speed 2.5G
2.5G shall be shown in the Speed field.
#### Previous command output (if the output of a command-line utility has changed)
Case1:
Ethernet is configured as 2.5G, but Speed 2G is shown.
admin@sonic:~$ show interfaces status Ethernet0
  Interface    Lanes    Speed    MTU    FEC                 Alias    Vlan    Oper    Admin    Type    Asym PFC
-----------  -------  -------  -----  -----  --------------------  ------  ------  -------  ------  ----------
  Ethernet0        2       2G   9100    N/A  twentyfivethousandE1  routed      up       up     N/A         N/A

Case2: 
The port speed for Ethernet0/Ethernet1 be configured as 100M, but only G are shown in the Speed field. 

admin@sonic:~$ show int sta Ethernet0-1
  Interface    Lanes    Speed    MTU    FEC                 Alias    Vlan    Oper    Admin    Type    Asym PFC
-----------  -------  -------  -----  -----  --------------------  ------  ------  -------  ------  ----------
  Ethernet0        2        G   9100    N/A  twentyfivethousandE1  routed      up       up     N/A         N/A
  Ethernet1        1        G   9100    N/A  twentyfivethousandE2  routed      up       up     N/A         N/A


#### New command output (if the output of a command-line utility has changed)
None
